pandas-stubs/core/dtypes/missing.pyi

```diff
@@ -1,12 +1,15 @@
 from typing import overload
 
 import numpy as np
+from numpy import typing as npt
 from pandas import (
     DataFrame,
     Index,
     Series,
 )
 
+from pandas._libs.missing import NAType
+from pandas._libs.tslibs.nattype import NaTType
```

Comment:
can you use `from pandas._libs.tslibs import NaTType` here?  That's what we have used elsewhere